### PR TITLE
Update to rely on HTML table rendering instead of images

### LINKED ISSUE #31: Update commented HTML tables to be rendered by LWT
Follows from https://github.com/CU-DBMI/set-website/pull/28#issuecomment-1904015299 and new version of LWT.

### PR DESCRIPTION
**Pull request by d33bs**
This PR addresses a prior issue with HTML rendering which involved static image workarounds (fixed in a recent release of LWT).

Closes #31

### CODE REVIEW

**Comment by d33bs** [comment]
Looks like the build fails based on a bundler and Ruby dependency challenge.

```text
7:53:32 AM: ERROR:  Error installing bundler:
7:53:32 AM: 	There are no versions of bundler (= 2.5.6) compatible with your Ruby & RubyGems
7:53:32 AM: 	bundler requires Ruby version >= 3.0.0. The current ruby version is 2.7.2.137.
```

**Comment by vincerubinetti** [comment]
[Netlify by default comes with Ruby 2.7 installed](https://docs.netlify.com/configure-builds/available-software-at-build-time/), and a recent template version (I think v1.2.0) updated Gem versions such that we need Ruby > 3.0. This is okay because Netlify hasn't been the recommended way to do PR previews since v1.0.0. The PR preview action seems to have worked fine.

I see that the post has some dangling `</table>` tags that weren't closed properly.

**Comment by vincerubinetti** [comment]
Playing around locally, it seems like the `<strong>` before the nested `<table>` is what's causing the issue. If I wrap the two in a `<div>`, such that the `<td>` only has one child, it works. This must be some bug in the markdown parser. At least it's not an issue with the template this time.

**Comment by vincerubinetti** [comment]
Turns out this is an issue with jekyll spaceship, very weird:

https://github.com/jeffreytse/jekyll-spaceship/issues/97

**Comment by vincerubinetti** [comment]
Unfortunately, it seems like jekyll-spaceship hasn't been updated in years and that bug is unlikely to be fixed. But it provides too much value to be disabled outright imo. So the `<div>` wrapper work-around would be recommended here. I would update it myself, but it's a fork and I don't think I have permissions. If you want to update this, we can merge and close.

**Comment by d33bs** [comment]
Thanks for the follow up, I tried to wrap the nested tables and saw that the rendering still looks a bit unusual. Maybe it's the multiple nested tables that's throwing it off? I can also close without merge here if it'd be best.

image

**Comment by vincerubinetti** [comment]
The opening `<div>` should go above the `<strong>`. The bug happens when a `<td>` has more than one child.

File: _posts/2024-01-22-Python-Memory-Management-and-Troubleshooting.md [hunk 1]
```diff
@@ -179,28 +180,32 @@ Please see [The Linux Programming Interface by Michael Kerrisk, Chapter 6.3: Mem
...
 <tr><td>📦</td><td>📦</td><td>📦</td></tr>
 </table>
+</div>
 
 </td>
 <td>
 
 <strong>D.)</strong> Two boxes are used, the other remains empty (ready for use).
 
+<div>
```
Comment (vincerubinetti) [inline _posts/2024-01-22-Python-Memory-Management-and-Troubleshooting.md:197]: This one's still here

Comment (d33bs) [reply _posts/2024-01-22-Python-Memory-Management-and-Troubleshooting.md]: Thanks!

**Comment by d33bs** [comment]
Thank you @vincerubinetti for your help with this! Merging this in.

**Pull request merged**